### PR TITLE
[MapRouletteUploadCommand] Add documentation

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

This adds some documentation on how to run the command the `MapRouletteUploadCommand`.

### Potential Impact:

N/A

### Unit Test Approach:

N/A

### Test Results:

N/A

### CODE REVIEW

File: docs/maproulette_upload.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* **checkinCommentPrefix** - Prefix added to default check in comment of [prefix ISO - CheckName]
+
+## Example
+
+The following command will upload EdgeCrossingEdge & SinkIsland checks to the `checks_example_project` Project on staging.maproulette.org. 
+Please remember to change the MapRoulette API key. You can find this in your MapRoulette user settings.
+
+```java
+java -cp build/libs/atlas-checks-6.1.3-SNAPSHOT-shaded.jar org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand \
+-maproulette="https://staging.maproulette.org:443:checks_example_project:api|key" \
```
Comment (atiannicelli) [inline docs/maproulette_upload.md:32]: I don't think you want "staging" in there. I had to remove it to make it work for me.
```suggestion
-maproulette="https://maproulette.org:443:checks_example_project:api|key" \
```

File: docs/maproulette_upload.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# MapRoulette Upload Command
+
+The [MapRouletteUploadCommand](/src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java) deserilizes line-delimited Atlas Check flags, and uploads them to the provided MapRoulette server.
+
+The command creates a new Challenge for each Atlas Check and country in the input data. For example: the following Challenges are created if the input data contains flags for EdgeCrossingEdgeCheck and SinkIslandCheck in USA and CAN:
+
+1. CAN - EdgeCrossingEdgeCheck
```
Comment (atiannicelli) [inline docs/maproulette_upload.md:7]: Not to be picky, but I think this creates "Canada - EdgeCrossingEdgeCheck"
```suggestion
1. Canada - EdgeCrossingEdgeCheck
```

File: docs/maproulette_upload.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+2. CAN - SinkIslandCheck
```
Comment (atiannicelli) [inline docs/maproulette_upload.md:8]:
```suggestion
2. Canada - SinkIslandCheck
```

File: docs/maproulette_upload.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+2. USA - EdgeCrossingEdgeCheck
```
Comment (atiannicelli) [inline docs/maproulette_upload.md:9]:
```suggestion
2. United States - EdgeCrossingEdgeCheck
```

File: docs/maproulette_upload.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+3. USA - SinkIslandCheck
```
Comment (atiannicelli) [inline docs/maproulette_upload.md:10]:
```suggestion
3. United States - SinkIslandCheck
```

File: docs/maproulette_upload.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+* **checkinComment** (optional) - Value of Changeset check in comment
+* **checkinCommentPrefix** - Prefix added to default check in comment of [prefix ISO - CheckName]
+
+## Example
+
+The following command will upload EdgeCrossingEdge & SinkIsland checks to the `checks_example_project` Project on staging.maproulette.org. 
+Please remember to change the MapRoulette API key. You can find this in your MapRoulette user settings.
+
+```java
+java -cp build/libs/atlas-checks-6.1.3-SNAPSHOT-shaded.jar org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand \
```
Comment (atiannicelli) [inline docs/maproulette_upload.md:31]: I had to copy https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.29/slf4j-api-1.7.29.jar to the same directory and add that to the class path. Is that worth putting in here or would java experts just know that?

**Review by atiannicelli — CHANGES_REQUESTED** [review]
Awesome. Thanks for putting this together. I was able to get it to work with a few small changes that I added comments for.

File: docs/maproulette_upload.md (hunk 6 above)
Comment (danielduhh) [reply docs/maproulette_upload.md:31]: Ah yeah.. sl4j is excluded in the gradle build. We could probably remove that.

**Comment by danielduhh** [comment]
Great review, thanks @atiannicelli !

File: docs/maproulette_upload.md (hunk 6 above)
Comment (atiannicelli) [reply docs/maproulette_upload.md:31]: OK, looks good as is then.

**Review by atiannicelli — APPROVED** [review]
Looks good!

File: docs/maproulette_upload.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+* **config** - Path to Atlas Checks configuration
+* **countries** (optional) - Comma delimited list of country permit list
+* **checks** (optional) - Comma delimted list of checks permit list
+* **outputPath** (optional) - Path to store text file of new challenge ids
+* **checkinComment** (optional) - Value of Changeset check in comment
+* **checkinCommentPrefix** (optional) - Prefix added to default check in comment of [prefix ISO - CheckName]
+
+## Example
+
+The following command will upload EdgeCrossingEdge & SinkIsland checks to the `checks_example_project` Project on staging.maproulette.org.
```
Comment (Bentleysb) [inline docs/maproulette_upload.md:27]:
```suggestion
The following command will upload EdgeCrossingEdge & SinkIsland checks to the `checks_example_project` Project on maproulette.org. 
```

**Review by Bentleysb — APPROVED** [review]

**Pull request merged**